PopulateTools/gobierto
# 716 - Previsualization of draft elements

Connects to #716 

### What does this PR do?

Permits admins to see draft elements in the frontend.

### How should this be manually tested?

Check both draft and published elements are visible in the frontend by clicking on their links from the admin view.

### Does this PR changes any configuration file?

This PR provides a rake task to regenerate admin preview tokens. This task should be executed periodically.
